CONTRIBUTING.md

```diff
@@ -12,6 +12,8 @@ Thanks for your interest in contributing to MongoDB Atlas Terraform Provider, th
   - [Running Acceptance Tests](#running-acceptance-tests)
```

Comment:
not for this PR but I'm feeling more and more the pain @gssbzn mentions about contributing file size :-) 
maybe at some moment we can have in this file the very basic stuff that people need to start contributing, and refer to other files for more specific details.

Reply:
yes makes sense, one option I see is to define a README.md in scaffolding folder and reference it from CONTRIBUTING.md file.